helics/_build.py

```diff
@@ -42,13 +42,13 @@
 # ffi.set_source(
 #     "_py_helics",
 #     f"""#include "{CHELICS}" """,
-#     libraries=["helicsSharedLib"],
+#     libraries=["libhelics"],
```

Comment:
is the lib supposed to be on this?

Reply:
Yes!

Reply:
why wasn't it on previously?

Reply:
I think I may have made the change locally but didn't push.